services/liberapay/liberapay-goal.service.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+const { InvalidResponse } = require('..')
+const { LiberapayBase } = require('./liberapay-base')
+const { colorScale } = require('../../lib/color-formatters')
+
+module.exports = class LiberapayGoal extends LiberapayBase {
+  static get route() {
+    return this.buildRoute('goal')
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'Liberapay goal progress',
+        namedParams: { entity: 'Changaco' },
+        staticPreview: this.render({ percentAcheived: 33 }),
+      },
+    ]
+  }
+
+  static render({ percentAcheived }) {
+    return {
+      label: 'goal progress',
+      message: `${percentAcheived}%`,
```

Comment:
Would decimal precision ever be a consideration here? Not sure whether the underlying data points could ever allow for an ugly decimal here or not

Reply:
We're already rounding it to the nearest int in `handle()`

Reply:
My bad! Missed that